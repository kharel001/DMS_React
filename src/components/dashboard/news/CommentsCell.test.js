import React from 'react';
import CommentsCell from './CommentsCell';
import {
    configure,
    shallow,
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})

const setup = (props = {}) => {
    return shallow( < CommentsCell {...props}  />
    )
}

const findByTestAttr=(wrapper, val)=>{
    return wrapper.find(`[data-test='${val}']`)
}


const dummy_props={
   data: {
    name:"name1", 
    desc:"Description", 
    image:"image1"
   }
}
describe("CommentsCell Comp",()=>{
it("Snapshot",()=>{
    const wrapper= setup(dummy_props);
    expect(wrapper).toMatchSnapshot()

})

})