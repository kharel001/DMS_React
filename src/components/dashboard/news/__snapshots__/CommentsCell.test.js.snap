// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentsCell Comp Snapshot 1`] = `ShallowWrapper {}`;
